opencti-platform/opencti-graphql/src/database/middleware.js

```diff
@@ -471,7 +472,7 @@ const loadElementsWithDependencies = async (context, user, elements, opts = {})
         const message = `From ${element.fromId} is ${validFrom}, To ${element.toId} is ${validTo}`;
         logApp.warn(`Auto delete of invalid relation ${element.id}. ${message}`);
         // Auto deletion of the invalid relation
-        await elDeleteElements(context, SYSTEM_USER, [element], storeLoadByIdWithRefs);
+        // await elDeleteElements(context, SYSTEM_USER, [element], storeLoadByIdWithRefs);
```

Comment:
Why remove this ?
There are some side effects ?

Reply:
Just forgot to remove the comment after some testing